CIP-????/README.md

```diff
@@ -0,0 +1,530 @@
+off-chain trading. Furthermore, thanks to the eUTxO model, transaction pieces have the same level of
+determinism as whole transactions.
+
+## Motivation: why is this CIP necessary?
+
+A key assertion underlying this CIP is **a batch of properly signed transaction pieces is
+functionally equivalent to a single transaction signed by all parties**. To highlight the truth of
+this assertion, consider two complementary atomic swap transaction pieces:
+
+- Alice wants to swap 10 ADA for 10 DJED so she creates a transaction piece with:
+    - 1 input from her address with 10 ADA
+    - 1 output to her address with 10 DJED
+- Bob wants to swap 10 DJED for 10 ADA so he creates a transaction piece with:
+    - 1 input from his address with 10 DJED
+    - 1 output to his address with 10 ADA
+
+Individually, neither of these transaction pieces are balanced transactions which means they can't
+be submitted to the blockchain. But if they are put together, the combination *is* a balanced
+transaction. Transaction pieces must be included all-or-nothing which means Alice's input can only
+be spent if the transaction actually includes her required output. 
+
+Given the all-or-nothing requirement, does it matter that Alice did not sign Bob's piece or the
+overall transaction? No, it doesn't because spending Bob's input doesn't require a signature from
+Alice, and she was guaranteed to get the output she wanted. The same is true for Bob's signature.
+There is no reason why the batching of these two transaction pieces should invalidate the
+signatures. Requiring Bob to also sign Alice's transaction piece is like requiring a smart contract
+to validate the spending of all UTxOs in a transaction, even those that aren't protected by that
+smart contract.
+
+Transaction pieces are the perfect model of an intent since *they are inherently flexible* (ie, a
+single piece can encompass multiple inputs, outputs, mints, etc) and *they can be unbalanced*. Being
+able to create unbalanced intents is critical to a healthy intent ecosystem because most intents are
+fundamentally only pieces of a transaction. Look back at Alice's and Bob's pieces above. The atomic
+swap intents don't say anything about the counter-party to the swap, and they shouldn't need to.
+Requiring users to specify who the counter-party should be fundamentally limits who can actually
+fulfill the intent. Without being able to create unbalanced transaction pieces, users would be
+forced to specify inputs and outputs for the potential counter-party just to finish balancing the
+transaction. 
+
+Another benefit of not having to specify counter-party information is that these transaction pieces
+can be trivially batched together without requiring coordination. As a result, a single transaction
+can trivially satisfy the intents of many complete strangers. In short, being able to properly model
+unbalanced intents makes transaction pieces a very natural and efficient intent settlement system.
+
+> [!IMPORTANT]
```

Comment:
```suggestion
> ⚠️ **Important**<br>
```
This looks great on GitHub there are sites derived from the CIP source material where this construction might not look very good, or at least will appear unstyled.  Currently no merged CIPs use this construction & you might be the first. 

Maybe you can tell from https://github.com/cardano-foundation/cf-cip-frontend how it would look (I searched for "admonition" on the source code & didn't find anything).  All things considered it might be suitable to leave these the way they are, but in general I think someday we will want CIPs with portable primitives (i.e. classic Markdown + the math extensions).

Reply:
@fallen-icarus I'm going to withdraw this suggestion since for some reason the CIP author community has started to converge upon GitHub admonitions recently; e.g.: https://github.com/cardano-scaling/CIPs/blob/decentralized-message-queue/CIP-0XXX/README.md#mithril-extra-network-usage

If & when it appears awkwardly on derived sites then I'll argue it's those sites' obligation to add some support for this GitHub syntax & display it intelligently.  Please keep an eye on how this looks on the Dev Portal and `cips.cardano.org` after it is merged & raise a red flag if the admonitions make a poor showing there. :)